libs/model/src/token/CreateToken.command.ts

```diff
@@ -0,0 +1,56 @@
+import { InvalidInput, type Command } from '@hicommonwealth/core';
+import * as schemas from '@hicommonwealth/schemas';
+import { ChainBase } from '@hicommonwealth/shared';
+import { models } from '../database';
+import { mustExist } from '../middleware/guards';
+
+export const CreateTokenErrors = {
+  TokenNameExists:
+    'The name for this token already exists, please choose another name',
+  InvalidEthereumChainId: 'Ethereum chain ID not provided or unsupported',
+  InvalidAddress: 'Address is invalid',
+  InvalidBase: 'Must provide valid chain base',
+  MissingNodeUrl: 'Missing node url',
+  InvalidNode: 'RPC url returned invalid response. Check your node url',
+};
+
+export function CreateToken(): Command<typeof schemas.CreateToken> {
+  return {
+    ...schemas.CreateToken,
+    auth: [],
```

Comment:
Add auth middleware here. I assume only community admins can perform this command?